scripts/minikube.sh

```diff
@@ -99,6 +99,10 @@ function minikube_losetup() {
     ${minikube} ssh 'sudo sh -c "rm -f /sbin/losetup && cp ~docker/losetup /sbin"'
 }
 
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+# shellcheck source=build.env
+[ ! -e "${SCRIPT_DIR}"/../build.env ] || source "${SCRIPT_DIR}"/../build.env
```

Comment:
I do not think this is helpful. Some CI jobs pass/change the environment to their needs before running the minikube.sh script. This will undo that again.

Reply:
won't it cause any issue as we need to install different kubernetes version for e2e and helm testing?

Reply:
yes, that is one example of what I am referring to

Reply:
Hmm, I see.
It's like the top-level script source build.env and then calls minikube.sh
I thought it will be good to make minikube.sh self-sufficient by loading the env variables within itself.